playbooks/preflight-check.yml

```diff
@@ -112,7 +112,7 @@
       when:
         - lxc_fs_check.stdout_lines
         - enable_lxc_fs_check | default(true) | bool
-      with_items: "{{ lxc_fs_check.stdout_lines }}"
+      with_items: "{{ lxc_fs_check.stdout_lines | default('LXC FS check is unavailable.')}}"
```

Comment:
Because of the requirement that `lxc_fs_check.stdout_lines` have a value on line 113, won't this default never be used?

Reply:
@rexredinger when 'enable_lxc_fs_check' flag was turned to 'false' manually by user/deployer to bypass lxc filesystem check, 'lxc_fs_check' variable won't be created since task 'Gather lxc filesystem used percentages' is skipped. Which makes following task 'Check if lxc filesystem is greater than 50% full' to use void variable 'lxc_fs_check'. This PR grants 'lxc_fs_check.stdout_lines' has alternative default value to avoid void variable using error. Though it probably never will be used.

Reply:
Thanks for the explanation! Can you change the default message to help explain that?  Something like "No LXC FS check was made."

Reply:
Yeah, which more makes sense.